package/MDAnalysis/topology/TXYZParser.py

```diff
@@ -0,0 +1,104 @@
+# -*- Mode: python; tab-width: 4; indent-tabs-mode:nil; coding:utf-8 -*-
+# vim: tabstop=4 expandtab shiftwidth=4 softtabstop=4
+#
+# MDAnalysis --- http://www.mdanalysis.org
+# Copyright (c) 2006-2017 The MDAnalysis Development Team and contributors
+# (see the file AUTHORS for the full list of names)
+#
+# Released under the GNU Public Licence, v2 or any higher version
+#
+# Please cite your use of MDAnalysis in published work:
+#
+# R. J. Gowers, M. Linke, J. Barnoud, T. J. E. Reddy, M. N. Melo, S. L. Seyler,
+# D. L. Dotson, J. Domanski, S. Buchoux, I. M. Kenney, and O. Beckstein.
+# MDAnalysis: A Python package for the rapid analysis of molecular dynamics
+# simulations. In S. Benthall and S. Rostrup editors, Proceedings of the 15th
+# Python in Science Conference, pages 102-109, Austin, TX, 2016. SciPy.
+#
+# N. Michaud-Agrawal, E. J. Denning, T. B. Woolf, and O. Beckstein.
+# MDAnalysis: A Toolkit for the Analysis of Molecular Dynamics Simulations.
+# J. Comput. Chem. 32 (2011), 2319--2327, doi:10.1002/jcc.21787
+#
+"""
+TXYZ topology parser
+====================
+
+Tinker topology parser: reads information from .txyz and .arc files.
+Atom types are read from column 6, while bond connectivity is read from column 7 onwards.
+see ../coordinates/TXYZ.py for further documentation about the Tinker format.
```

Comment:
Restructured text allows you the create links. Do not refer to Python files, that's not helpful in the online docs. In this case write 
```rest
Tinker_ topology parser .....
....
see :mod:`MDAnalysis.coordinates.TXYZ` for further documentation...

.. _Tinker: http:/:dasher.wustl.edu/tinker
```

(Check the URL, I wrote it from memory. ). In this way you generate links inside the docs and to other resources on the web.